services/src/main/java/org/keycloak/broker/oidc/mappers/AbstractJsonUserAttributeMapper.java

```diff
@@ -125,9 +125,9 @@ public void preprocessFederatedIdentity(KeycloakSession session, RealmModel real
 		}
 		attribute = attribute.trim();
 
-		String value = getJsonValue(mapperModel, context);
+		Object value = getJsonValue(mapperModel, context);
 		if (value != null) {
-			context.setUserAttribute(attribute, value);
+			context.setUserAttribute(attribute, value.toString());
```

Comment:
@hmlnarik This method is inherited by several "social" user attribute mappers. I wonder if it we need to check if the value is a list since the context has an overloaded method to accept a list of attribute values or if we don't ever need to handle this case in the sub-classes.